my-app/src/utils/Api.js

```diff
@@ -14,7 +17,7 @@ export const getSavedArticles = (token) => {
     method: "GET",
     headers: {
       "Content-Type": "application/json",
-      authorization: `Bearer: ${token}`,
+      authorization: `Bearer ${token}`,
     },
   }).then(checkResponse);
```

Comment:
**Could be improved**

You can make a special method (function ) for fetching and checking responses **not to duplicate** it in every request:

```js

  function request(url, options) {
    return fetch(url, options).then(checkResponse)
  }

```

And now you can safely replace `fetch` with `request` and remove repeated checking. And you don’t need to change anything else.